src/app/home/content/product-list/product-list.component.html

```diff
@@ -1,27 +1,32 @@
-<div class="container ">
-  
-    <div class="row results-base">
-      <div class="col-12 col-md-3 col-sm-3 col-lg-3 product-base" *ngFor="let product of products.products | filter : selectedTaxonIds; trackBy: trackByFn"
-        [style.margin]="getMargin()" itemscope itemtype="https://schema.org/Product">
-        <div class="product-thumbShim"></div>
-        <a itemprop="url" [routerLink]="['/', product.slug]">
-          <img itemprop="image" class="product-thumb" alt="{{product.name}}" [src]="getProductImageUrl(product.master.images[0]?.product_url)">
-          <div class="product-productMetaInfo">
-            <div class="morechoices">
-              More Choices Available
-            </div>
-            <div itemprop="name" class="product-brand">{{ product.name }}</div>
-            <div itemprop="offers" itemscope itemtype="https://schema.org/Offer" class="product-price">
-              <span>
-                <span itemprop="price" class="product-discountedPrice">&#8377; {{ product.price }}</span>
+<div class="container "><div class="row results-base">
+    <div class="col-12 col-md-3 col-sm-3 col-lg-3 product-base" *ngFor="let product of products.products | filter : selectedTaxonIds; trackBy: trackByFn"
+      [style.margin]="getMargin()" itemscope itemtype="https://schema.org/Product">
+
+      <div class="product-thumbShim"></div>
```

Comment:
Remove this `<div class="product-thumbShim"></div>`
Please format document.(indentation)

Reply:
Removed